smrf/framework/model_framework.py

```diff
@@ -234,14 +234,14 @@ def loadTopo(self):
 
     def create_distribution(self):
         """
-        This initializes the distirbution classes based on the configFile
+        This initializes the distribution classes based on the configFile
```

Comment:
Spelling is not my strong siut.

Reply:
I let my IDE do that for me :wink:

Reply:
This was pre good IDE's when that was written...